MissionTable.cpp

```diff
@@ -26,6 +26,15 @@ void WriteBoolCell(bool boolean, std::streamsize cellWidthInChars);
 
 std::string ConvertMissionTypeToString(MissionTypes missionType);
 
+constexpr std::array<std::string_view, 7> columnTitles{
```

Comment:
I wouldn't have thought to use string_view in the context of storing in a array. I wonder how that stores the underlying data, as a char* or as a std::string?

Reply:
The raw `const char*` null terminated C-strings from the literals are stored in the global or read-only section of the executable. Either way, their lifetime is the lifetime of the program. The `std::string_view` instances in the array are then references to the the `const char*` literals along with their length.